consumers/hidden-test-consumer/src/hidden_test_consumer/hidden_test_process_pb2_grpc.py

```
@@ -0,0 +1,97 @@
+
+if _version_not_supported:
+    raise RuntimeError(
+        f'The grpc package installed is at version {GRPC_VERSION},'
+        + f' but the generated code in hidden_test_process_pb2_grpc.py depends on'
+        + f' grpcio>={GRPC_GENERATED_VERSION}.'
+        + f' Please upgrade your grpc module to grpcio>={GRPC_GENERATED_VERSION}'
+        + f' or downgrade your generated code using grpcio-tools<={GRPC_VERSION}.'
+    )
+
+
+class HiddenTestProcessStub(object):
```

Ruff (0.8.2)

Class `HiddenTestProcessStub` inherits from `object`

Remove `object` inheritance

(UP004)